screens/ChatScreen.js

```diff
@@ -145,31 +160,37 @@ class ChatScreen extends Component {
     this.setState(previousState => ({
       messages: localMessages.concat(previousState.messages),
     }));
+
+    this.cacheMessages();
+  }
+
+  async cacheMessages() {
```

Comment:
It would be great if the load logic could load the message before a user even got to the chat screen, like every time the app is launched we can sync new messages

Reply:
Yeah completely agree. I'm planning to refactor the twilio logic into a service in the future, and at that point I will add more complex message loading logic to handle message caching from the moment you open the app.